packages/theming/android-theme/README.md

```diff
@@ -0,0 +1,5 @@
+# Android-theme
+
+Code and definitions for creating an Android Theme for FluentUI React Native.
+
+The theme follows color, typography, spacing and other values to closely match [FluentUI Android](https://github.com/microsoft/fluentui-android/tree/master/FluentUI/src/main/res/values). The theme is work in progress and changes are expected.
```

Comment:
It's probably fine to just leave this link as the base GitHub repo (https://github.com/microsoft/fluentui-android) but that's a nitpick and I wouldn't consider it necessary for check in.